Tools/environment_install/install-ROS-ubuntu.sh

```diff
@@ -209,6 +209,16 @@ heading "${green}Initializaing rosdep${reset}"
 sudo rosdep init || true
 # To find available packages, use:
 rosdep update
+
+# Installing ardupilot_gazebo
+git clone https://github.com/SwiftGust/ardupilot_gazebo
```

Comment:
shouldn't this be "ArduPilot" instead of "SwiftGust"?

Reply:
Sir the ArduPilot's package for ardupilot_gazebo doesn't have rover model

Reply:
let's add it!